apps/test/unit/code-studio/components/progress/ScriptTeacherPanelTest.jsx

```diff
@@ -181,4 +184,99 @@ describe('ScriptTeacherPanel', () => {
       expect(wrapper.find('StudentTable')).to.have.length(0);
     });
   });
+  describe('SelectedStudentInfo', () => {
```

Comment:
nit: move these tests into their own file called `SelectedStudentInfoTest.jsx` and add empty newlines between `describe` blocks

Reply:
ignore the first part of that nit -- i just noticed you already have a `SelectedStudentInfoTest.jsx` file and these tests should stay in `ScriptTeacherPanelTest.jsx`!